src/ldclient_eval.erl

```diff
@@ -40,8 +40,8 @@ target_match
 }.
 
 -type all_flags_state_options() :: #{
-    with_reasons => boolean()
-%%    client_side_only => boolean(), % TODO: Support.
+    with_reasons => boolean(),
+    client_side_only => boolean()
```

Comment:
It seemed reasonable to implement this at the same time, and that allows the contract tests to be passed as well.